ci-operator/config/openshift-knative/serving/openshift-knative-serving-release-next__411.yaml

```diff
@@ -1,253 +1,72 @@
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/helloworld
   to: knative-serving-test-helloworld
 - dockerfile_path: openshift/ci-operator/knative-test-images/httpproxy/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/httpproxy
   to: knative-serving-test-httpproxy
 - dockerfile_path: openshift/ci-operator/knative-test-images/pizzaplanetv1/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/pizzaplanetv1
   to: knative-serving-test-pizzaplanetv1
 - dockerfile_path: openshift/ci-operator/knative-test-images/pizzaplanetv2/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/pizzaplanetv2
   to: knative-serving-test-pizzaplanetv2
 - dockerfile_path: openshift/ci-operator/knative-test-images/readiness/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/readiness
   to: knative-serving-test-readiness
 - dockerfile_path: openshift/ci-operator/knative-test-images/runtime/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/runtime
   to: knative-serving-test-runtime
 - dockerfile_path: openshift/ci-operator/knative-test-images/servingcontainer/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/servingcontainer
   to: knative-serving-test-servingcontainer
 - dockerfile_path: openshift/ci-operator/knative-test-images/sidecarcontainer/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/sidecarcontainer
   to: knative-serving-test-sidecarcontainer
 - dockerfile_path: openshift/ci-operator/knative-test-images/singlethreaded/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/singlethreaded
   to: knative-serving-test-singlethreaded
 - dockerfile_path: openshift/ci-operator/knative-test-images/timeout/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/timeout
   to: knative-serving-test-timeout
 - dockerfile_path: openshift/ci-operator/knative-test-images/volumes/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/volumes
   to: knative-serving-test-volumes
 - dockerfile_path: openshift/ci-operator/knative-test-images/wsserver/Dockerfile
-  from: base
-  inputs:
-    test-bin:
-      paths:
-      - destination_dir: .
-        source_path: /go/bin/wsserver
   to: knative-serving-test-wsserver
 promotion:
   additional_images:
-    knative-serving-src: src
-  name: knative-nightly
+    serving-src: src
```

Comment:
Renamed. This change broke eventing on last release. Do we want to try to go with it or should I add a hack for serving (e.g. https://github.com/openshift-knative/hack/blob/main/pkg/prowgen/prowgen_config.go#L160)